app/pages/cliniciandetails/cliniciandetails.js

```diff
@@ -199,7 +219,8 @@ class ClinicianDetails extends React.Component {
     const form = [
       { type: 'name', name: 'firstName', label: 'first name', value: formValues.firstName },
       { type: 'name', name: 'lastName', label: 'last name', value: formValues.lastName },
-      { type: 'clinicalRole', name: 'clinicalRole', label: 'clinical role', value: formValues.clinicalRole }
+      { type: 'clinicalRole', name: 'clinicalRole', label: 'clinical role', value: formValues.clinicalRole },
+      { type: 'name', name: 'country', label: 'country', value: formValues.country },
```

Comment:
there should be a condition based on "config.ALLOW_SELECT_COUNTRY" here, or a default. With the current code when config.ALLOW_SELECT_COUNTRY=false the form is considered invalid